ext_mongodb.php

```diff
@@ -620,6 +626,94 @@ class BulkWriteException extends WriteException {}
 /* {{{ BSON and Serialization Classes */
 namespace MongoDB\BSON;
 
+/* {{{ Interfaces */
+interface BinaryInterface
+{
+	public function serialize() : string;
+	public function unserialize(mixed $data) : void;
+
+	public function __construct(string $data, int $type);
+	public function getType();
+	public function getData() : string;
+	function __debugInfo() : array;
+}
+
+interface Decimal128Interface
+{
+	public function serialize() : string;
+	public function unserialize(mixed $data) : void;
+	
+	function __construct(string $decimal);
+	function __toString() : string;
+	function __debugInfo() : array;
+}
+
+interface JavascriptInterface
+{
+	public function serialize() : string;
+	public function unserialize(mixed $data) : void;
+	
+	public function __construct(string $code, ?mixed $scope = NULL);
+	function __debugInfo() : array;
```

Comment:
`getCode()`, `getScope()`, and `__toString()` (returns a string) should probably be included here.